stress/shell/src/main/java/alluxio/stress/cli/suite/MaxThroughput.java

```diff
@@ -73,7 +73,9 @@ public MaxThroughputSummary runSuite(String[] args) throws Exception {
 
     List<String> baseArgs = new ArrayList<>(Arrays.asList(args));
 
-    prepareBeforeAllTests(baseArgs);
+    if (mParameters.mSkipPrepare) {
```

Comment:
i think you meant `!mParameters.mSkipPrepare` here

Reply:
Exactly, Thanks for remind me